### ISSUE TITLE
The attribute `fit` is not implemented.

### ISSUE THREAD

**Issue by yangchenyun** [PolymerElements/paper-header-panel#89]
The doc mentions the `fit` attribute will apply `--layout-fit` to the `#maincontent`, but this promise is not implemented here.  

Would either mention it in the doc to ask the user imports the `iron-positioning` in iron-flex-layout-classes.html or we put it in the stylesheet in scope `:host ::content .fit { ... }`

**Comment by blasten** [comment]
That's true. We should either remove that from the docs or add that behavior back to the element. PR welcome.
